### DISCUSSION TITLE
import bim file into powerbi fabric

### DISCUSSION THREAD

**Discussion by janhoran** [TabularEditor/TabularEditor3#1228] [Learn]
I am trying to see what conditions i need to meet in fabric to import my model.bim. I also know that I need to change the connection properties in my bim file, because I am going to connect not thru ODBC but "natively" to databricks to read the data from and load it into the tabular model. Possibly firewall needs to be opened to fabric from my laptop where I am trying to deploy the model into fabric.

**Comment by janhoran** [comment]
Please share if you have any experience with this as I don't. One thing is to import the bim file and another to make it work. I am at the very beginning of this.

**Reply by otykier** [reply]
I would recommend creating a sample model using Fabric / Power BI Desktop, which imports just a few tables from Databricks, then using two instances of Tabular Editor, compare how the two models are configured in terms of data sources, partitions, etc. You can then also copy/paste objects between those two instances, to quickly populate one from the other.

**Reply by janhoran** [reply]
That is a good idea, thanks Daniel!
